src/main/webapp/script.js

```diff
@@ -30,62 +30,100 @@ async function getYouTubeComments() {
 }
 
 /**
- * Fetches data and adds to html
+ *  Fetches data and adds to html.
  */
 async function getChart() {
   $('form').submit(async function() {
     document.getElementById('loading-img').style.display = 'block';
     commentStats = await getYouTubeComments();
-    averageScore = commentStats.averageScore;
-    averageMagnitude = commentStats.averageMagnitude;
     sentimentBucketList = commentStats.sentimentBucketList;
+    wordFrequencyMap = commentStats.wordFrequencyMap;
 
-    const CommentSentimentTable = new google.visualization.DataTable();
-    CommentSentimentTable.addColumn('number', 'InclusiveStart');
-    CommentSentimentTable.addColumn('string', 'SentimentRange');
-    CommentSentimentTable.addColumn('number', 'CommentCount');
+    getBarChart(sentimentBucketList);
```

Comment:
Can we rename these get*Charts to communicate their intent more clearly?

I.e.
`displaySentimentBucketChart`
and 
`displayWordCloudChart`